services/drupal/web/modules/custom/epa_metrics/src/ViewTimer.php

```diff
@@ -75,15 +75,10 @@ public static function stop(ViewExecutable $view) {
 
     // The statsd protocol uses milliseconds, so we have to convert from the seconds that
     // microtime(TRUE) returns.
-    foreach ([
-               'view' => 'Drupal.Views.' . $id,
-               'build' => 'Drupal.ViewsBuild.' . $id,
-               'execute' => 'Drupal.ViewsExecute.' . $id,
-               'render' => 'Drupal.ViewsRender.' . $id] as $stage => $timingkey) {
-      if (isset($stats[$stage])) {
-        $service->timing($timingkey, $stats[$stage] * 1000);
-      }
-    }
+    $service->timing('Drupal.Views.' . $id, $stats['view'] * 1000);
+    $service->timing('Drupal.ViewsBuild.' . $id, $stats['build'] * 1000);
+    $service->timing('Drupal.ViewsExecute.' . $id, $stats['execute'] * 1000);
+    $service->timing('Drupal.ViewsRender.' . $id, $stats['render'] * 1000);
     $service->flush();
```

Comment:
I was trying to avoid a problem I was seeing where some views were coming back missing some of these stats, so we were getting php notices when trying to access some of the array keys.

Reply:
Oops, that's my bad. I thought it was something that had been swept up in the merge conflict resolution. I'll restore it.

Reply:
Actually, it doesn't look like it's in stable yet, so let's get it there first.

Reply:
It'll hit stable when EPAD8-884 lands.

Reply:
And it's in master because EPAD8-884 has already been merged to master.

Reply:
Gotcha. I've restored the refactor.